fastlane/lib/fastlane/actions/appetize.rb

```diff
@@ -100,6 +104,11 @@ def self.available_options
                                        description: "Target url of the zipped build. Either this or `path` must be specified",
                                        is_string: true,
                                        optional: true),
+          FastlaneCore::ConfigItem.new(key: :platform,
+                                       env_name: "APPETIZE_PLATFORM",
+                                       description: "Platform. Either `ios` or `android`. Default is `ios`",
+                                       is_string: true,
+                                       optional: true),
```

Comment:
Instead of making this value optional, you can set `ios` as the `default_value:`